fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -1023,13 +1096,9 @@ private SqlQueryData processTokenParm(QueryParameter queryParm, String tableAlia
                     }
                     whereClauseSegment.append(tableAlias + DOT).append(CODE_SYSTEM_ID).append(operator)
```

Comment:
shouldn't this be split into two appends? `.append(PARAMETER_TABLE_ALIAS).append(DOT)`

Reply:
fixed